main.py

```diff
@@ -126,24 +150,46 @@ def main(
                 continue
 
             # Paste the resized rubbish object onto the background at the determined position
-            output_image[
-                paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width
-            ] = copy_paste_without_blend(
-                output_image[paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width],
-                rubbish_object_resized,
-                rubbish_mask_resized,
-            )
+            if save_mode == SAVE_WITHOUT_ALPHA_BLEND:
+                output_image[
+                    paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width
+                ] = copy_paste_without_blend(
+                    output_image[paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width],
+                    rubbish_object,
+                    rubbish_mask,
+                )
+            elif save_mode == SAVE_WITH_ALPHA_BLEND:
+                output_image[
+                    paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width
+                ] = alpha_blend(
+                    output_image[paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width],
+                    rubbish_object, 
+                    rubbish_blurred_mask
+                )
+            else:
+                print("Invalid save mode chosen. Please choose between 'alpha_blend' and 'without_alpha_blend'.")
+
 
             # Generate mask for each pasted rubbish onto new background
             mask_for_rubbish = np.zeros_like(background, dtype=np.uint8)
             mask_for_rubbish[
                 paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width
             ] = copy_paste_without_blend(
                 mask_for_rubbish[paste_y: paste_y + scaled_height, paste_x: paste_x + scaled_width],
-                np.ones_like(rubbish_object_resized) * 255,  # White mask for pasted rubbish
-                rubbish_mask_resized,
+                np.ones_like(rubbish_object) * 255,  # White mask for pasted rubbish
+                rubbish_mask,
             )
 
+            cv2.namedWindow('View output_image', cv2.WINDOW_NORMAL)
```

Comment:
Why are we showing the output image? When generating 100 images it will slow down the process.

Reply:
Forgot to delete those lines. Fixing it now.